packages/erc3k/contracts/ERC3000Data.sol

```diff
@@ -52,6 +57,7 @@ library ERC3000Data {
     }
 
     function hash(Payload memory payload) internal pure returns (bytes32) {
+        // Note: I'd try to follow the same order defined in the struct to be consistent
```

Comment:
Good call. The order in the struct is kinda random rn tbf, we should reorder for clarity